coverage/.resultset.json

```diff
@@ -1,21 +1,40 @@
 {
   "RSpec": {
     "coverage": {
-      "/Users/joechirchirillo/Desktop/3_Mod/projects/consultancy/poke_gochi_api/app/models/trainer.rb": {
+      "/Users/delarosa/Desktop/Mod 3/Projects/poke-gochi-API/app/models/trainer.rb": {
```

Comment:
This is interesting, wonder why it changes names depending on the user and if it has any significant impact. I'll look into this and report the findings.